cpp/velox/shuffle/VeloxShuffleWriter.cc

```diff
@@ -376,8 +386,145 @@ arrow::Status VeloxShuffleWriter::partitioningAndDoSplit(facebook::velox::RowVec
   return arrow::Status::OK();
 }
 
-arrow::Status VeloxShuffleWriter::stop() {
+arrow::Status VeloxShuffleWriter::doSort(facebook::velox::RowVectorPtr rv, int64_t memLimit) {
+  currentInputColumnBytes_ += rv->estimateFlatSize();
+  batches_.push_back(rv);
```

Comment:
QQ: looks like this only enqueue the batches, the sort is done in Celeborn side?

Reply:
@zhouyuan In the HashPartitioner's compute, it will encapsulate the information of each row belonging to the same PartitionId in the RowVector into an int, and put it into the Vector of the map, which is equivalent to replacing the Sort with a HashMap.